catalog/transformer/catalog-transformer-streaming-api/src/main/java/org/codice/ddf/transformer/xml/streaming/Gml3ToWkt.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.transformer.xml.streaming;
+
+import java.io.InputStream;
+
+public interface Gml3ToWkt {
+
+    /**
+     * Convert a GML3 XML snippet String to a WKT String
+     *
+     * @param xml String of GML XML
+     * @return a WKT String representation of the input
```

Comment:
The impl returns `String` or `null`, so these javadocs are incorrect. Presumably the record would fail to ingest if you just let the exception bubble up, but I'm ok with that, since it's presumably invalid metadata at that point.